alerting/provider/ntfy/ntfy.go

```diff
@@ -36,7 +38,11 @@ func (provider *AlertProvider) IsValid() bool {
 	if provider.Priority == 0 {
 		provider.Priority = DefaultPriority
 	}
-	return len(provider.URL) > 0 && len(provider.Topic) > 0 && provider.Priority > 0 && provider.Priority < 6
+	tokenValid := true
+	if len(provider.Token) > 0 {
+		tokenValid = strings.HasPrefix(provider.Token, "tk_")
+	}
+	return len(provider.URL) > 0 && len(provider.Topic) > 0 && provider.Priority > 0 && provider.Priority < 6 && tokenValid
```

Comment:
call me nitpicky, but `tokenValid` should be something like `isTokenValid` instead.